pkg/commands/command_helper.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2020-Present VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package commands
+
+import "C"
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/pivotal/build-service-cli/pkg/k8s"
+)
+
+type CommandHelper struct {
+	dryRun         bool
+	outputResource bool
+	wait           bool
+
+	outWriter io.Writer
+	errWriter io.Writer
+
+	objectPrinter k8s.ResourcePrinter
+	builder       strings.Builder
+}
+
+func NewCommandHelper(cmd *cobra.Command) (*CommandHelper, error) {
+	dryRun, err := getBoolFlag("dry-run", cmd)
+	if err != nil {
+		return nil, err
+	}
+
+	output, err := getStringFlag("output", cmd)
+	if err != nil {
+		return nil, err
+	}
+
+	wait, err := getBoolFlag("wait", cmd)
+	if err != nil {
+		return nil, err
+	}
+
+	if err != nil {
+		return nil, err
+	}
+
+	var resourcePrinter k8s.ResourcePrinter
+
+	outputResource := len(output) > 0
+	if outputResource {
+		resourcePrinter, err = k8s.NewObjectPrinter(output)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	return &CommandHelper{
+		dryRun,
+		outputResource,
+		wait,
+		cmd.OutOrStdout(),
+		cmd.ErrOrStderr(),
+		resourcePrinter,
+		strings.Builder{},
+	}, nil
+}
+
+func (cp CommandHelper) IsDryRun() bool {
+	return cp.dryRun
+}
+
+func (cp CommandHelper) CanWait() bool {
```

Comment:
maybe `Wait()` or `ShouldWait()` ?

Reply:
Since it is returning a boolean and not exactly waiting.. leaning towards ShouldWait (or CanWait)

Reply:
went with ShouldWait()